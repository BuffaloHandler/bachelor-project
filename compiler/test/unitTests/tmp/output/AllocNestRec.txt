14
20
0
1
1
532
ret 0
