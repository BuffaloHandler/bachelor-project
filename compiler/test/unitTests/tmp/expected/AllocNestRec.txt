14
20
0
1
1
532
